inc/slalevel_ticket.class.php

```diff
@@ -214,7 +214,8 @@ static function doLevelForTicket(array $data, $sltType) {
                   // Put next level in todo list
                   $next = $slalevel->getNextSltLevel($ticket->fields[$sltField],
                                                      $data['slalevels_id']);
-                  $slt->addLevelToDo($ticket, $next);
+                  $current = $data['slalevels_id'];
+                  $slt->addLevelToDo($ticket, $next, $current);
```

Comment:
True, this doesn't solves anything. 
I misplaced to edit line 214 in this file. made another commit bf03283